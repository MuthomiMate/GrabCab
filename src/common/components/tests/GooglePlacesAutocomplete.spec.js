import React from "react";
import { shallow } from "enzyme";
import GooglePlaces from "../GooglePlacesAutoComplete";

describe("<GooglePlaces />", () => {

    const testProps = {
        placeholder: "Where From ?",
        id: "from"
    }

    it("renders without crashing", () => {
        const wrapper = shallow(<GooglePlaces {...testProps}/>);
        expect(wrapper).toMatchSnapshot();
    })
})