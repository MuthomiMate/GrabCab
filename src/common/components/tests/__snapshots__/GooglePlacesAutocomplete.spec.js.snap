// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GooglePlaces /> renders without crashing 1`] = `
<GooglePlacesAutocomplete
  GooglePlacesSearchQuery={
    Object {
      "rankby": "distance",
      "types": "food",
    }
  }
  GoogleReverseGeocodingQuery={Object {}}
  autoFillOnNotFound={false}
  autoFocus={false}
  currentLocation={false}
  currentLocationLabel="Current location"
  debounce={200}
  editable={true}
  enableEmptySections={true}
  enableHighAccuracyLocation={true}
  enablePoweredByContainer={true}
  fetchDetails={true}
  filterReverseGeocodingByTypes={Array []}
  getDefaultValue={[Function]}
  isRowScrollable={true}
  keyboardShouldPersistTaps="always"
  listViewDisplayed={false}
  minLength={2}
  nearbyPlacesAPI="GooglePlacesSearch"
  numberOfLines={1}
  onFail={[Function]}
  onNotFound={[Function]}
  onPress={[Function]}
  onSubmitEditing={[Function]}
  onTimeout={[Function]}
  placeholder="Where From ?"
  placeholderTextColor="#A8A8A8"
  predefinedPlaces={Array []}
  predefinedPlacesAlwaysVisible={false}
  query={
    Object {
      "components": "country:ke",
      "key": "AIzaSyDZCD7tYrunw72Zg093q23IYB9AM1cU4YQ",
      "language": "en",
    }
  }
  renderRightButton={[Function]}
  returnKeyType="search"
  styles={
    Object {
      "description": Object {
        "color": "white",
        "fontWeight": "bold",
      },
      "listView": Object {
        "zIndex": 500,
      },
      "predefinedPlacesDescription": Object {
        "color": "white",
      },
      "textInput": Object {
        "backgroundColor": "#0A0F10",
        "color": "white",
      },
      "textInputContainer": Object {
        "backgroundColor": "#0A0F10",
      },
    }
  }
  suppressDefaultStyles={false}
  textInputHide={false}
  textInputProps={Object {}}
  timeout={20000}
  underlineColorAndroid="transparent"
/>
`;
